packages/@sanity/desk-tool/src/panes/document/documentPanel/DocumentPanel.tsx

```diff
@@ -64,7 +64,7 @@ export const DocumentPanel = function DocumentPanel(props: DocumentPanelProps) {
   const requiredPermission = value._createdAt ? 'update' : 'create'
   const liveEdit = useMemo(() => Boolean(getSchemaType(documentType)?.liveEdit), [documentType])
   const docPermissionsInput = useMemo(() => {
-    return {...value, _id: liveEdit ? 'dummy-id' : 'drafts.dummy-id'}
+    return {...value, _id: liveEdit ? value._id : `drafts.${value._id}`}
```

Comment:
we have some helpers to ensure IDs if you think it's appropriate to use them

```
import {getPublishedId, getDraftId} from 'part:@sanity/base/util/draft-utils'
```

Reply:
ah yes, you can import it from `import {getPublishedId} from '@sanity/base/_internal'`

Reply:
Makes sense, I've updated the code to reflect that 👍